jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BaseImageSpec.java

```diff
@@ -29,8 +29,8 @@
  * <pre>{@code
  * image: gcr.io/example/baseimage
  * platforms:
- *   - see {@link PlatformSpec}
- *   - see {@link PlatformSpec}
+ *   - see }{@link PlatformSpec}{@code
+ *   - see }{@link PlatformSpec}{@code
```

Comment:
This Javadoc looks strange to me. Which error prone warnings is this addressing?

Reply:
A warning is given where another javadoc tag(i.e. @link) is included inside @code.
I referred to the link below.  
https://stackoverflow.com/questions/8208032/is-nesting-of-javadoc-tag-codelink-allowed

Reply:
Got it. Thanks.

Reply:
Interesting. Looks good then.